professor2-ppc64-flag-change.patch

```diff
@@ -20,7 +20,7 @@ index adf3b19..fabdbf0 100644
                  depends=glob("../include/*.h"),
                  include_dirs=[incdir, os.path.join(srcdir, "pyext", "professor2")],
 -                extra_compile_args="-std=c++11 -O3 -Wno-unused-but-set-variable -Wno-sign-compare".split(),
-+                extra_compile_args="-std=c++11 -O3 -mcpu=native -Wno-unused-but-set-variable -Wno-sign-compare".split(),
++                extra_compile_args="-std=c++20 -O3 -mcpu=native -Wno-unused-but-set-variable -Wno-sign-compare".split(),
```

Comment:
@iarspider , may be for cpp std we just patch the Makefile using perl/sed comamnd e.g.
```
#Make sure the default c++sdt stand is c++11 in Makefile
grep -q 'std=c++11' Makefile
#Change c++ std to %{cms_cxx_standard}
sed  -e 's|-std=c++11|-std=c++%{cms_cxx_standard}|' Makefile
```